llama_hub/library.json

```diff
@@ -622,5 +622,23 @@
       "bucket",
       "storage"
     ]
+  },
+  "NewsArticleReader": {
+    "id": "news",
```

Comment:
id needs to be `web/news` and `web/rss_news` (to show up on llamahub UI)

Reply:
this seems to be broken